packages/compile-solidity-tests/test/compileWithPragmaAnalysis.ts

```diff
@@ -129,8 +133,11 @@ describe("compileWithPragmaAnalysis", function () {
         options: config,
         paths: [path.join(sourceDirectory, "withImports", "C.sol")]
       });
+      if (!compilations) {
```

Comment:
(There is still this which I think should be changed to use an `assert.isDefined` or something rather than `if (...) { assert.fail() }`, but I think that's the last one.)